src/modules/wrap-with-usememo.ts

```diff
@@ -0,0 +1,36 @@
+import { selectedText, importMissingDependencies, activeFileName, allText, selectedTextStart, selectedTextEnd } from "../editor";
+import { replaceSelectionWith } from "../code-actions";
+import { buildEffectHook, buildUseCallbackHook, buildUseMemo } from "../snippet-builder";
+import { codeToAst, astToCode } from "../parsing";
+import { persistFileSystemChanges } from "../file-system";
+import * as t from '@babel/types';
+import { pathContains } from "../ast-helpers";
+import traverse from "@babel/traverse";
+
+
+function isFunction(node) {
+    return t.isFunctionDeclaration(node) || t.isFunctionExpression(node) || t.isArrowFunctionExpression(node)
+}
+
+export function isVariableDeclarationWithNonFunctionInit(text) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [expected parameter: 'text' to have a typedef](https://app.codacy.com/gh/wix/vscode-glean/pullRequest?prid=5535229)